R/hmde_model_pars.R

```diff
@@ -102,6 +102,6 @@ hmde_vb_multi_ind_pars <- function(){
 hmde_linear_single_ind_pars <- function(){
   list(measurement_pars_names = c("y_hat"),
        individual_pars_names = c("ind_beta_0", "ind_beta_1"),
-       error_pars_names = c("global_error_sigma"),
+       error_pars_names = c(NULL),
```

Comment:
this branch adds a vignette, but there's also some change here to R functions and Stan files. are these intended?

Reply:
Yes, this is the vignette for the linear single ind model, and because I changed the structure of the model to provide a simpler vignette there are structural changes within it.